docs/supported_languages.rst

```diff
@@ -0,0 +1,125 @@
+   interpreted as described in `RFC 2119`_.
+
+Thresholds for Translation and Review Coverage
+----------------------------------------------
+
+.. list-table::
+   :widths: 30 30 30
+   :header-rows: 1
+
+   * -
+     - Translation Coverage (of Source Strings)
+     - Review Coverage (of Translations) [#review_coverage]_
+   * - **Initial coverage,** for a language to be granted support
+     - 100%
+     - 100%
+   * - **Ongoing coverage,** for a language to remain supported
+     - 80%
+     - 100%
+
+In addition to these thresholds, the SecureDrop team will:
+
+#. always prioritize the translation of source-facing strings, given their
+   importance for sources' security; and
+
+#. inform Localization Lab when particular strings should be prioritized or
+   even considered blocking for a given release.
+
+Granting Support for a Language
+-------------------------------
+
+Granting support for a new language consists of adding an entry in the
+``supported_locales`` object in ``securedrop``'s ``i18n.json`` and/or in the
+"Localization" section in ``securedrop-client``'s ``MANIFEST.in``.  Other steps,
+such as communication, are at the discretion of the Localization Manager.
+
+#. A language *L* that reaches coverage in time for a release
+   version *V* SHOULD be nominated for support in version *V*.
+
+#. The Localization Manager SHOULD ask Localization Lab whether they
+   believe *L*'s `language team`_ is likely to be able to maintain coverage for
+   the foreseeable future.
+
+        #. If so, the Localization Manager SHOULD grant support for *L*.
+
+        #. If not, the Localization Manager MUST NOT grant support for *L*.
+
+Revoking Support for a Language
+-------------------------------
+
+Revoking support for a currently-supported language consists of removing the
+language's entries in ``i18n.json`` and/or ``MANIFEST.in``.  Other steps, such
+as communication, are at the discretion of the Localization Manager.
+
+Consider an expected release timeline as follows:
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Version
+     - Date
+   * - V1
+     - January 1
+   * - V2
+     - March 1
+   * - V3
+     - May 1
+
+Then:
+
+#. A language *L* that misses coverage for a release version *V1* MUST be
+   considered on probation for up to the next two releases *V2* and *V3*.
+   While on probation, a language is still considered supported until it has
+   missed coverage for a total of 3 consecutive release/localization cycles.
+
+        #. In consultation with Localization Lab, the Localization
+           Manager MAY consult the `language census`_ and reach out to
+           administrators who may be able to contribute to translation and
+           review.
+
+#. If *L* misses coverage again for *V2* and does not regain
+   coverage for *V3*, then the Localization Manager SHOULD revoke support for
+   *L* for *V3*.
```

Comment:
I think this should be V4, no? We say in the preceding paragraph that a language may be be considered supported if it has missed coverage for a total of 3 cycles.

If so, I think we should define V4 in the table above as well.

Reply:
Hmm.  There's a counting nuance here.  In the scenario as written:

1. *V1*: coverage drops below threshold → begins probation
2. *V2*: coverage still below threshold → continues probation
3. *V3*: coverage still below threshold (at close of QA/localization period) → dropped

= 3 consecutive cycles.

In the scenario you're proposing:

1. *V1*: coverage drops below threshold → begins probation
2. *V2*: coverage still below threshold → continues probation
3. *V3*: coverage still below threshold → continues probation
4. *V4*: coverage still below threshold (at close of QA/localization period) → dropped

= 4 consecutive cycles.

Would this help clarify?

```rst
#. If *L* misses coverage again for *V2* and does not regain
   coverage by the end of the release/localization cycle for *V3*, then the Localization Manager SHOULD revoke support for
   *L* for *V3*.
```

Reply:
What confused me is the wording in the preceding paragraph: 

> While on probation, a language is still considered supported until it has missed coverage for a total of 3 consecutive release/localization cycles.

My interpretation was that a `release/localization cycle` ends _with the software being released_, so that the 3 consecutive cycles imply three consecutive releases.

If the intended meaning is that at the time of translation merge _during_ a release process, the language was merged with insufficient support three consecutive times, then putting it in those or similar terms that avoid the ambiguity of "consecutive release/localization cycles" would help, I think.